minecraft_servers/suchtcraft/manifest.json

```diff
@@ -0,0 +1,36 @@
+{
+  "server_name": "suchtcraft",
+  "nice_name": "SuchtCraft",
+  "direct_ip": "SuchtCraft.net",
+  "social": {
+    "web": "https://suchtcraft.net/",
+    "web_support": "https://suchtcraft.net/forum/",
+    "discord": "https://suchtcraft.net/discord",
+    "twitter": "SuchtCraftNET",
+    "instagram": "suchtcraftnet"
+  },
+  "gamemodes": {
+    "stadt": {
+      "name": "Stadt",
+      "color": "#5454FC"
+    },
+	"challenges": {
+      "name": "Challenges",
+      "color": "#5454FC"
+    }
+  },
+  "discord": {
+    "server_id": 538780470192766986,
+    "rename_to_minecraft_name": true
```

Comment:
Please set to "false", as this is the default value.

Reply:
Changed with 1f74679de7ddc16a9434201b6b636a6c464ce71a